### PR TITLE
557: add integration test for threshold-calculation component

### PR DESCRIPTION
**Pull request by trbmcginnis**
- Add integration test for `components/transportation/threshold-calculation.js` 
- 8 computed properties in this component are tested (100% of lines in file)

Part of #557 testing effort

### CODE REVIEW

File: frontend/tests/integration/components/transportation/threshold-calculation-test.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          {
+            name: "Lots",
+            type: "lots",
+            spaces: 100
+          }
+        ],
+      })
+    }
+
+  test('it renders', async function(assert) {
```
Comment (godfreyyeung) [inline frontend/tests/integration/components/transportation/threshold-calculation-test.js]: nit: Title could use a bit more detail

File: frontend/tests/integration/components/transportation/threshold-calculation-test.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    await render(hbs`{{transportation/threshold-calculation analysis=transportationModel}}`);
+
+    // sumOfRatios = residentialUnitsRatio + officeSqFtRatio + regionalRetailSqFtRatio + localRetailSqFtRatio +
+    // restaurantSqFtRatio + communityFacilitySqFtRatio + offStreetParkingSpacesRatio
+    // sumOfRatios = 7.014399999999999, this is rounded to 7.01 in computed property `detailedAnalysisPopupText`
+    assert.ok(this.element.querySelector('[data-test-popup="detailed analysis"]').textContent.includes('7.01'));
+
+    // ## We are testing 7 computed properties in this test that are calculated using
+    // the transportation analysis model's computed property thresholdFor
+    // thresholdFor(type) { return this.get(`trafficZoneThresholds.${type}.zone${this.trafficZone}`); }
```
Comment (godfreyyeung) [inline frontend/tests/integration/components/transportation/threshold-calculation-test.js]: nit: a little confusing to have the function pasted here. wasn't sure at first if it was meant to be in this test

File: frontend/tests/integration/components/transportation/threshold-calculation-test.js [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+    );
+    let analysis = await project.get('transportationAnalysis');
+
+    this.set('transportationModel', analysis);
+
+    await render(hbs`{{transportation/threshold-calculation analysis=transportationModel}}`);
+
+    // sumOfRatios = residentialUnitsRatio + officeSqFtRatio + regionalRetailSqFtRatio + localRetailSqFtRatio +
+    // restaurantSqFtRatio + communityFacilitySqFtRatio + offStreetParkingSpacesRatio
+    // sumOfRatios = 7.014399999999999, this is rounded to 7.01 in computed property `detailedAnalysisPopupText`
```
Comment (godfreyyeung) [inline frontend/tests/integration/components/transportation/threshold-calculation-test.js]: nice comments!! 🔥 🔥

File: frontend/tests/integration/components/transportation/threshold-calculation-test.js (hunk 2 above)
Comment (allthesignals) [reply frontend/tests/integration/components/transportation/threshold-calculation-test.js]: yeah please delete uncommented code

**Pull request merged**